src/styles/blocks/card.scss

```
@@ -0,0 +1,87 @@
+.card {
+  width: $card-width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
```

You already have this in html tag
Remove in all cases
```suggestion
    line-height: 18px;
```